src/components/radioButton/RadioButton.js

```diff
@@ -133,9 +159,9 @@ class RadioButton extends BaseComponent {
     return value === selectedValue;
   }
 
-  getContainerStyle() {
+  getCircleStyle() {
```

Comment:
`circle` is a little bit abstract, you can call it radio button since it's the actual radio button. 
before when it was `container` it was enough since all this component rendered is a radio button.

Reply:
But this is only the outer circle, not the whole radio button